public/images/delete-site/export-content.svg

```diff
@@ -0,0 +1 @@
+<svg width="158" height="174" viewBox="0 0 158 174" xmlns="http://www.w3.org/2000/svg"><title>figure</title><g transform="translate(0 -5)" fill="none" fill-rule="evenodd"><rect fill="#D5E5EB" x="57" y="157" width="45" height="22" rx="4"/><text font-family="Open Sans" font-size="11" font-weight="526" fill="#FFF"><tspan x="69.736" y="172">.ZIP</tspan></text><path d="M89.5 131.5h-6v-9h-9v9h-6L79 142l10.5-10.5zm-21 13.5v3h21v-3h-21z" fill="#D5E5EB"/><path d="M61 118h36v36H61v-36z"/><path d="M.03 40.992l54.486-29.92 25.18 13.238-54.62 29.734L.032 40.992z" fill="#D5E5EB"/><path d="M157.9 40.992l-54.484-29.92-25.18 13.238 54.62 29.734L157.9 40.992z" fill="#D5E5EB"/><path d="M118.9 43.846c0 21.098-17.186 38.266-38.304 38.266-21.118 0-38.303-17.168-38.303-38.266S59.478 5.58 80.596 5.58 118.9 22.748 118.9 43.846zM65.55 74.81L49.132 29.837a34.013 34.013 0 0 0-2.992 14.008c0 13.624 7.952 25.367 19.41 30.963zM101.2 53.24c1.495-4.783 2.65-8.2 2.65-11.147 0-4.23-1.54-7.175-2.864-9.48-1.71-2.82-3.378-5.21-3.378-8.072 0-3.16 2.394-6.108 5.772-6.108l.47.042c-6.114-5.594-14.278-9.052-23.256-9.052-12.012 0-22.614 6.19-28.77 15.502l2.223.042c3.59 0 9.148-.427 9.148-.427 1.88-.086 2.094 2.605.256 2.86 0 0-1.88.214-3.976.3l12.568 37.284 7.525-22.593-5.344-14.692c-1.88-.085-3.633-.298-3.633-.298-1.84-.128-1.625-2.947.214-2.86 0 0 5.685.425 9.063.425 3.633 0 9.19-.427 9.19-.427 1.838-.086 2.094 2.605.215 2.86 0 0-1.84.214-3.934.3l12.44 36.985 3.422-11.446zM92.01 76.304c-.085-.172-.17-.3-.213-.47l-10.603-29L70.85 76.86c3.12.895 6.37 1.365 9.746 1.365 4.02 0 7.865-.683 11.414-1.92zm19.024-45.44c0 3.5-.643 7.43-2.608 12.342L97.91 73.57c10.216-5.936 17.098-17.04 17.098-29.724 0-5.98-1.495-11.616-4.188-16.528a31.01 31.01 0 0 1 .214 3.546z" fill="#A8BECE"/><path d="M133.263 53.938v37.715L79 113.728V76.014l54.263-22.076zM24.737 53.938v37.715L79 113.728V76.014L24.737 53.938z" fill="#D5E5EB"/><path d="M155.74 70.355L102.527 92.48 78.125 75.12l55.24-21.407 22.374 16.642z" stroke="#FFF" stroke-width="2.4" fill="#D5E5EB"/><path d="M2.08 70.355L55.294 92.48l24.402-17.36-54.582-21.297L2.08 70.355z" stroke="#FFF" stroke-width="2.4" fill="#D5E5EB"/></g></svg>
```

Comment:
@mtias: I suppose we could, although I'm not sure necessarily any big benefit of doing so. Having that image tag makes the component a bit more readable to me, but we still could definitely put it inline.